olp-cpp-sdk-dataservice-read/test/unit/src/CatalogClientTest.cpp

```diff
@@ -2375,16 +2461,15 @@ TEST_P(CatalogClientMockTest, GetCatalogCacheWithUpdate) {
   waitForEnd->get_future().get();
 
   // Request 2 to check there is a cached value.
-  EDGE_SDK_LOG_TRACE_F("CatalogClientMockTest", "Request Catalog, CacheOnly"); 
-  request.WithFetchOption(CacheOnly); future =
-    catalogClient->GetCatalog(request);
+  EDGE_SDK_LOG_TRACE_F("CatalogClientMockTest", "Request Catalog, CacheOnly");
```

Comment:
Hmm. You should not use logs in tests. Use SCOPED_TRACE("....") to see it only when test fails.